OrbitLinuxTracing/TracerThread.h

```diff
@@ -79,8 +80,10 @@ class TracerThread {
   // before switching to another one.
   static constexpr int32_t ROUND_ROBIN_POLLING_BATCH_SIZE = 5;
 
-  static constexpr uint64_t SMALL_RING_BUFFER_SIZE_KB = 256;
-  static constexpr uint64_t BIG_RING_BUFFER_SIZE_KB = 2048;
+  static constexpr uint64_t CONTEXT_SWITCHES_RING_BUFFER_SIZE_KB = 256;
+  static constexpr uint64_t UPROBES_RING_BUFFER_SIZE_KB = 8192;
```

Comment:
I have a feeling we'll need more than this.  8MB can hold about 125 uprobe events.  Seems to me that we'll hit the limit pretty quick once we instrument a few functions.  I'd crank that up to 32MB per core, and possibly reduce the size of the stack buffer we copy for each event.

Reply:
I'm happy to increase the limit, I followed your 32MB suggestion. As per reducing the size of the stack buffer, as we've discussed in the past I'm not in favor of that because that would reduce the hard limit on the size of stacks that Orbit supports.